src/cli/controller.js

```diff
@@ -285,8 +285,8 @@ class Controller {
    * @param {string} name The name of the function to call.
    * @param {object} data The data to send to the function.
    */
-  call (name, data) {
-    return this.client.callFunction(name, data);
+  call (name, data, resource, auth) {
```

Comment:
Instead of adding two new positional arguments, let's add a single `opts` positional argument on which you can attach any "keyword" arguments you need, such as `resource` and `auth`. This will make this method more future-proof.